civitai-downloader/src/api_client.py

```
@@ -0,0 +1,300 @@
+"""CivitAI API client implementation."""
+
+import asyncio
+import json
```

Ruff (0.12.2)

`json` imported but unused

Remove unused import: `json`

(F401)